cmd/pipeline/deploy.go

```diff
@@ -124,6 +126,7 @@ func deploy(ctx context.Context) *cobra.Command {
 	if err := cmd.Flags().MarkHidden("filename"); err != nil {
 		oktetoLog.Infof("failed to mark 'filename' flag as hidden: %s", err)
 	}
+	cmd.Flags().StringArrayVarP(&flags.labels, "label", "", []string{}, "set an environment label (can be set more than once)")
```

Comment:
In the `pipeline list` I am using this description:

```
"tag and organize dev environments using labels (multiple --label flags accepted)"
```

Which I suggested at the time for `preview list`. I think it helps from a product stand point to understand what `--label` is meant for.